apps/studio/prisma/seed.ts

```diff
@@ -217,13 +217,13 @@ async function main() {
       .values({ content: jsonb(PAGE_BLOB) })
       .returning("id")
       .executeTakeFirstOrThrow()
-    blobId = id
+    blobId = BigInt(id)
   }
 
   await db
     .insertInto("Resource")
     .values({
-      mainBlobId: blobId,
+      mainBlobId: String(blobId),
```

Comment:
question(non-blocking): just wanted to confirm, when inserting we should immediately use string here, rather than passing in the bigint?

Reply:
Yea, the generatedType for bigint is actually string. So kysely will expect a string to be passed in for this. However, the insert op will then convert this string into a bigint